App.tsx

```diff
@@ -1,13 +1,22 @@
 import {StyleSheet, SafeAreaView} from 'react-native';
 import React from 'react';
-import {FlatListToDoList, ToDoList} from './src/screens/toDoList';
+import {createStackNavigator} from '@react-navigation/stack';
+import {FlatListToDoList} from './src/screens/toDoList';
+import {NavigationContainer} from '@react-navigation/native';
+import {Login, LoginParam} from './src/screens/login';
+
+const Stack = createStackNavigator<LoginParam>();
 
 export default function App(): React.JSX.Element {
   return (
-    <SafeAreaView style={styles.container}>
-      <ToDoList />
-      {/* <FlatListToDoList /> */}
-    </SafeAreaView>
+    <NavigationContainer>
+      <SafeAreaView style={styles.container}>
+        <Stack.Navigator initialRouteName="Login">
+          <Stack.Screen name="Login" component={Login} />
+          <Stack.Screen name="FlatListToDoList" component={FlatListToDoList} />
+        </Stack.Navigator>
+      </SafeAreaView>
+    </NavigationContainer>
```

Comment:
Create a navigation folder inside src, where the navigation should be. The App.tsx should be clean.